vllm/entrypoints/openai/protocol.py

```diff
@@ -63,6 +63,8 @@ class ChatCompletionRequest(BaseModel):
     # Ordered by official OpenAI API documentation
     # https://platform.openai.com/docs/api-reference/chat/create
     messages: List[Dict[str, str]]
+    tools: Optional[List[Dict[str, Union[str, Dict]]]] = None
+    documents: Optional[List[Dict[str, str]]] = None
```

Comment:
Hi, 

I wonder which version of openai library support this "documents" parameter?